src/gui/core/placer.cpp

```diff
@@ -16,26 +16,26 @@
 #define GETTEXT_DOMAIN "wesnoth-lib"
 
 #include "gui/core/placer.hpp"
-
-#include <cassert>
 #include "gui/core/placer/horizontal_list.hpp"
 #include "gui/core/placer/vertical_list.hpp"
+#include "string_enums/grow_direction.hpp"
```

Comment:
This file has to be included by placer.hpp anyway, no need for it here.

While it's trivial, I'm currently looking at the dependency graphs while tiebreaking where the files should go.

Reply:
Fixed.